src/store/ShareStore.js

```diff
@@ -1,7 +1,10 @@
-import html2canvas from 'html2canvas';
 import { observable, action, reaction, computed, when } from 'mobx';
 import MenuStore from './MenuStore';
 import { downloadFileInBrowser, findAncestor } from './utils';
+import { loadScript } from '../utils';
+
+const html2canvasCDN = 'https://html2canvas.hertzen.com/dist/html2canvas.min.js';
```

Comment:
I couldn't find a CDN that has the latest version of html2canvas. Older versions don't render all the icons in chart controls.

Reply:
Wait actually nvm that. Dev ops have us covered. I’ll update the cdn link next week